### PR TITLE
committing

### PR DESCRIPTION
**Pull request by Samantha-KY**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    font-size: large;
+    font-weight: 100;
+}
+.intro__body{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 3rem;
+    margin-top: 2rem;
+}
+.intro__body__card{
```
Comment (mbukeRepo) [inline style.css]: Try to avoid nesting elements.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    background: #ffffff;
+    border: 1px solid #e8e8e8;
+    width: 328px;
+    height: 193px;
+}
+.intro__body__card__el{
```
Comment (mbukeRepo) [inline style.css]: Try to avoid nesting.

**Review by mbukeRepo — CHANGES_REQUESTED** [review]

**Review by mbukeRepo — APPROVED** [review]
LGTM

File: style.css [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    display: flex; 
+    gap: 2rem;
+}
+.nav__list__side{
+    display: flex;
+    gap: 1rem;
+    margin-top: 1rem;
+    margin-right: 6rem;
+}
+.body{
```
Comment (ritha-kankindi) [inline style.css]: It's better to style body at the top of your css. (nesting of blocks and elements)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+}
+.nav__list{
+    display: flex; 
+    gap: 6rem;
+}
+.nav__list__line{
+    display: flex; 
+    gap: 2rem;
+}
+.nav__list__side{
```
Comment (ritha-kankindi) [inline style.css]: Write this element right element format. like **nav__list-side** or if it is a modifier write **nav__list--side**

File: style.css [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+    .nav__title{
+        display: flex; 
+        gap: 5rem; 
+        margin-left: 5rem;
+    }
+    .nav__list{
+        display: flex; 
+        gap: 3rem;
+    }
+    .nav__list__line{
```
Comment (ritha-kankindi) [inline style.css]: Change this also

File: index.html [hunk 6]
```diff
@@ -99,10 +99,10 @@ <h1>WHY CHOOSE US</h1>
         class="intro__body"
       >
         <div
-         class="intro__body__card"
```
Comment (ritha-kankindi) [inline index.html]: Use right way of writing elements.  hint : intro__body-card or intro__body---card if it is a modifier

**Review by ritha-kankindi — APPROVED** [review]
Good Job

**Review by karemarsy — COMMENTED** [review]
I can't see your html file, I don't know why

**Comment by Samantha-KY** [comment]
Could you use this link; I don't know why it is not visible.
https://github.com/Samiskina/BEMClone/pull/1/commits/e6da14d60f760e571085311d131425b68026050f

**Review by karemarsy — APPROVED** [review]

File: index.html [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+    <div class="intro">
+      <h1>WHY CHOOSE US</h1>
+      <p class="intro__text">
+        Problems trying to resolve the conflict between the two major realms of
+        Classical physics: Newtonian mechanics
+      </p>
+      <div class="intro__body">
+        <div class="intro__card">
+          <div class="intro__card-el">
+            <div class="intro__card-el1 intro__body--display">
```
Comment (karemarsy) [inline index.html:55]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+            </div>
+            <p>
+              the quick fox jumps over the lazy dog the quick fox jumps over the
+              lazy dog
+            </p>
+          </div>
+        </div>
+        <div class="intro__card">
+          <div class="intro__card-el">
+            <div class="intro__card-el1 intro__body--display">
```
Comment (karemarsy) [inline index.html:67]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
```
Comment (karemarsy) [inline index.html:79]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
```
Comment (karemarsy) [inline index.html:91]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
```
Comment (karemarsy) [inline index.html:103]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
```
Comment (karemarsy) [inline index.html:115]: `intro__body--display` is included without it's default Element (intro__body).

File: index.html [hunk 13]
```diff
@@ -0,0 +1,238 @@
...
+          </div>
+          <div>
+            <h4>Get In Touch</h4>
+            <img src="/BEMClone/images/footerListSvg.svg" alt="">
+            <p>Lorem impsum dolor amit</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer__text">
```
Comment (karemarsy) [inline index.html:232]: `footer__text` is positioned outside the Block (footer).

File: index.html [hunk 14]
```diff
@@ -0,0 +1,238 @@
...
+      <p class="footer__parag footer__parag--color">
```
Comment (karemarsy) [inline index.html:233]: `footer__parag` is positioned outside the Block (footer).
`footer__parag--color` is positioned outside the Block (footer).

**Pull request merged**